LICENSE

```diff
@@ -1,21 +1,10 @@
 ## Licensing Information
 
-This library (UniDoc) has a dual license, a commercial one suitable for closed source projects and an
-AGPL license that can be used in open source software.
+This software package is a commercial product and requires a license
+code to operate.
 
-Depending on your needs, you must choose one of them and follow its policies. A detail of the policies
-and agreements for each license type are available in the [LICENSE.COMMERCIAL](LICENSE.COMMERCIAL)
-and [LICENSE.AGPL](LICENSE.AGPL) files.
-
-In brief, purchasing a license is mandatory as soon as you develop activities
-distributing the UniDoc software inside your product or deploying it on a network
-without disclosing the source code of your own applications under the AGPL license.
-These activities include:
-
- * offering services as an application service provider or over-network application programming interface (API)
- * creating/manipulating documents for users in a web/server/cloud application
- * shipping UniDoc with a closed source product
-
-Please see [pricing](http://unidoc.io/pricing) to purchase a commercial license or contact sales at sales@unidoc.io
-for more info.
+The use of this software package is governed by the end-user license agreement
+(EULA) available at: https://unidoc.io/eula/
 
+To obtain a Trial license code to evaluate the software, please visit
```

Comment:
Update to get a free metered code

Reply:
updated at 8bd35aeb42ce72eb86b57c7481271c1a95750b86